index.js

```diff
@@ -3656,6 +3656,43 @@
       [Fn(a, b), m(a), m(a)],
       Z.sortBy);
 
+  //# zipWith :: (a -> b -> c) -> Array a -> Array b -> Array c
+  //.
+  //. Creates a new array out of the two supplied arrays
+  //. by applying the given function to each equally-positioned
+  //. pair of elements from each given array. The returning
+  //. array is truncated to the length of the shorter of the two
+  //. input arrays.
+  //.
+  //. ```javascript
+  //. > S.zipWith(S.add, [1, 2, 3], [3, 2, 1])
+  //. [4, 4, 4]
```

Comment:
Let's drop this example. It's clear from the following examples what should happen when the arrays are of equal length.